pom.xml

```diff
@@ -13,7 +12,15 @@
     <modelVersion>4.0.0</modelVersion>
 
     <properties>
+        <javax-json.version>1.1</javax-json.version>
+        <jackson-databind.version>2.9.4</jackson-databind.version>
+        <jboss-logging.version>3.3.2.Final</jboss-logging.version>
+        <junit.version>4.12</junit.version>
+        <logmanager.version>2.1.8.Final</logmanager.version>
+        <maven-compiler-plugin.version>3.8.0</maven-compiler-plugin.version>
         <resteasy.version>4.0.0.CR1</resteasy.version>
+        <resteasy-tracing-api.version>1.0.0.Beta4</resteasy-tracing-api.version>
+        <undertow.ver>1.4.18.Final</undertow.ver>

```

Comment:
For all the dependencies here, why can't we rely on what RESTEasy would transitively pull? That would reduce the burden of maintaining versions here in future.

Reply:
yes sir!